src/main/java/org/reaktivity/command/log/internal/LogCommand.java

```diff
@@ -37,33 +37,30 @@ public static void main(String[] args) throws Exception
         options.addOption(builder("t").hasArg().required(false).longOpt("type").desc("streams* | counters").build());
         options.addOption(builder("d").longOpt("directory").hasArg().desc("configuration directory").build());
         options.addOption(builder("v").longOpt("verbose").desc("verbose output").build());
-        options.addOption(builder("bc").longOpt("bufferCheck").desc("buffer capacity check").build());
+        options.addOption(builder("qd").longOpt("queue-depths").desc("display queue depths").build());
 
         CommandLine cmdline = parser.parse(options, args);
 
+        boolean verbose = cmdline.hasOption("verbose");
+        Properties properties = new Properties();
+        String directory = cmdline.getOptionValue("directory");
+        properties.setProperty(Configuration.DIRECTORY_PROPERTY_NAME, directory);
+
+        final Configuration config = new LogCommandConfiguration(properties);
 
         if (cmdline.hasOption("help") || !cmdline.hasOption("directory"))
         {
             HelpFormatter formatter = new HelpFormatter();
             formatter.printHelp("log", options);
         }
+        else if (cmdline.hasOption("queue-depths"))
+        {
+            new LogQueueDepthCommand(config, System.out::printf, verbose).invoke();
+        }
```

Comment:
Remove this and use `type` argument with value `queues` to trigger use of `LogQueueDepthCommand`.